presage/src/manager/mod.rs

```diff
@@ -9,7 +9,7 @@ use std::fmt;
 
 pub use self::confirmation::Confirmation;
 pub use self::linking::Linking;
-pub use self::registered::{ReceivingMode, Registered, RegistrationData, RegistrationType};
+pub use self::registered::{Received, Registered, RegistrationData, RegistrationType};
```

Comment:
Maybe it makes send to export it from the location where it is actually defined, that is, from `crate::model::messages::Received`?